1 basics

FedorRasputin opened the pull request.

Commit:
commit

Commit:
1

purpleschoolofficial commented on src/App.vue:
```diff
@@ -0,0 +1,9 @@
+</template>
+
+<script setup>
+const date = new Date();
```
Рекомендуется использовать форматирование даты для более удобного отображения пользователю. Например, можно использовать `date.toLocaleDateString()` или `date.toLocaleString()`, чтобы дата выглядела привычнее.

purpleschoolofficial replied:
Если требуется, чтобы дата обновлялась в реальном времени (например, при переходе на новую дату без перезагрузки страницы), можно использовать реактивное свойство и таймер. Сейчас дата фиксируется при загрузке компонента.

Commit:
style: add global CSS variables and Montserrat font

This commit introduces a set of global CSS variables for consistent theming and adds the Montserrat font to the project. The changes improve the overall styling and maintainability of the application.
